Lib/glyphsLib/builder/axes.py

```diff
@@ -157,8 +159,16 @@ def to_designspace_axes(self):
         ):
             axis_wanted = True
 
+        # See https://github.com/googlefonts/glyphsLib/issues/568
+        if custom_mapping and axis.tag in custom_mapping:
```

Comment:
You are correct. If a mapping isn't defined in the Axis Mapping, it will use the previous mechanisms. I agree with your omit approach. I'll change this and remove the copy pasta.